bibliography.bib

```diff
@@ -7,6 +7,26 @@ @inproceedings{gullasch2011cache
   organization={IEEE}
 }
 
+@incollection{ristenpart2009cloud,
+  title={Hey, you, get off of my cloud: exploring information leakage in third-party compute clouds},
+  author={Ristenpart, Thomas and Tromer, Eran and Shacham, Hovav and Savage, Stefan},
+  booktitle={CCS '09 Proceedings of the 16th ACM conference on Computer and communications security},
+  pages={199--212},
+  year={2009},
+  publisher={ACM},
+  annote={
+    \setlength{\parskip}{1.5ex}
+    Describes a method for achieving co-residence with target VMs on the cloud and exploiting information leakage between these VMs using Amazon's EC2 service as an example. The attack takes advantage of the fact that many VMs can be created by the same user, and that there is a possibility that these might run on the same hardware as a target due to the way the VMs share physical resources. Once co-location is achieved, an attacker can use side-channels (namely caches) to get information about the other instances running on the same machine as the malicious VM.  
```

Comment:
Editing will be way easier if you stick to 120-character lines and place each sentence on a new line.